pkg/networkservice/common/refresh/client_test.go

```diff
@@ -302,6 +325,82 @@ func TestRefreshClient_NoRefreshOnFailure(t *testing.T) {
 	require.Never(t, cloneClient.validator(2), testWait, testTick)
 }
 
+func TestRefreshClient_CalculatesShortestTokenTimeout(t *testing.T) {
+	t.Cleanup(func() { goleak.VerifyNone(t) })
+
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	testData := []struct {
+		Chain                     []time.Duration
+		ExpectedRefreshTimeoutMax time.Duration
+		ExpectedRefreshTimeoutMin time.Duration
+	}{
+		{
+			Chain:                     []time.Duration{time.Hour},
+			ExpectedRefreshTimeoutMax: 20*time.Minute + time.Second,
+			ExpectedRefreshTimeoutMin: 20*time.Minute - time.Second,
```

Comment:
Please use const time range.
```suggestion
			ExpectedRefreshTimeout: 20*time.Minute
```

Reply:
Apply this for other places.

Reply:
Done